README.md

```diff
@@ -6,14 +6,23 @@
 
 ## Fulltext Search on Firebase with Meilisearch
 
-npm install -g firebase-tools
+### To run this project in local
 
-firebase login
+`npm install -g firebase-tools`
 
-firebase --open-sesame extdev
+`firebase login`
 
-npm run install
+`firebase --open-sesame extdev`
 
-npm run build
+You maybe need to install typescript in global to run this project<br>
+`npm install typescript -g`
 
-firebase emulators:start
+`npm run install`
```

Comment:
I would suggest to rename this script to something else as `install` because it can be confusing. 
To install packages in a repository, you do `npm install` or `yarn`. But here, in your repository, doing `npm install` will create a `package-lock.json` at root, which is not what you want (you want to install the dependencies inside the `function` directory).

Reply:
You right! I will rename it